src/testdrive/src/main.rs

```diff
@@ -200,8 +200,27 @@ async fn main() {
     let mut errors = Vec::new();
     let mut error_files = Vec::new();
 
-    if files.is_empty() {
-        files.push("-".to_string())
+    let mut files = vec![];
+
+    if args.globs.is_empty() {
+        files.push(PathBuf::from("-"))
+    } else {
+        let walker = match GlobWalkerBuilder::from_patterns(".", &args.globs).build() {
```

Comment:
Should this path only trigger if there are any wildcard characters being used? If no wildcard characters, just assume literal file names and use them verbatim, do not walk anything.